lua/cmp_path/init.lua

```diff
@@ -128,6 +149,7 @@ source._candidates = function(_, context, dirname, offset, callback)
               filterText = '/' .. name,
               insertText = '/' .. name,
               kind = cmp.lsp.CompletionItemKind.File,
+	      path = dirname .. '/' .. name,
```

Comment:
Indentation.


And You should add custom data to `CompletionItem.data` field (it was defined in LSP spec).

Reply:
ok